### ISSUE TITLE
Single SAM-SA-2-V-759-5V23 reports as having 4 missiles by default

### ISSUE THREAD

**Issue by walder** [Auranis/HighDigitSAMs#11]
While writing unit tests for Skynet, I noticed that a single SAM-SA-2-V-759-5V23 launcher reports as initially having 4 missiles:

```
	lu.assertEquals(launcher1:getTypeName(), "S_75M_Volhov_V759")
	lu.assertEquals(launcher1:getRange(), 56000)
	lu.assertEquals(launcher1:getMaximumFiringAltitude(), 30000)
	lu.assertEquals(launcher1:getInitialNumberOfMissiles(), 4)
```

getAmmo() returns:

```
{
    {
        count=4,
        desc={
            Nmax=5,
            RCS=0.39669999480247,
            _origin="",
            altMax=30000,
            altMin=100,
            box={
                max={x=4.7303376197815, y=0.84564626216888, z=0.84564626216888},
                min={x=-5.8387970924377, y=-0.84564626216888, z=-0.84564626216888}
            },
            category=1,
            displayName="SA2V759",
            fuseDist=50,
            guidance=4,
            life=2,
            missileCategory=2,
            rangeMaxAltMax=56000,
            rangeMaxAltMin=45000,
            rangeMin=6000,
            typeName="SA2V759",
            warhead={caliber=500, explosiveMass=302, mass=302, type=1}
        }
    }
}
```
I believe the correct number should be 1 for this type of  launcher.

**Comment by walder** [comment]
The same also applies to the 5p73 V-601P ln launcher.

**Comment by Auranis** [comment]
Fixed locally. The count of 4 missiles is correct for the 5p73, I think, but I'll double check that it works correctly.

**Comment by Auranis** [comment]
Fixed in v1.4.1
